src/main/java/com/amazonaws/services/kinesis/leases/impl/LeaseCoordinator.java

```diff
@@ -437,10 +438,10 @@ public boolean updateLease(T lease, UUID concurrencyToken)
      * @param maximumPoolSize Maximum allowed thread pool size
      * @return Executor service that should be used for lease renewal.
      */
-    private static ExecutorService getLeaseRenewalExecutorService(int maximumPoolSize) {
+    public static ExecutorService getDefaultLeaseRenewalExecutorService(int maximumPoolSize) {
```

Comment:
Does this need to be public? Seems like this is only being used in the Worker, so making it package private would be better.

Reply:
Worker is in a different package (com.amazonaws.services.kinesis.clientlibrary.lib.worker) than LeaseCoordinator (com.amazonaws.services.kinesis.leases.impl). We would need to keep it public.